src/olympia/stats/tests/test_views.py

```diff
@@ -734,7 +743,7 @@ def test_no_status_page(self):
             reverse('stats.statuses.beta', args=[self.addon.slug])
 
         with self.assertRaises(NoReverseMatch):
-            reverse('tats.statuses_series.beta', args=[self.addon.slug])
+            reverse('stats.statuses_series.beta', args=self.series_args)
```

Comment:
Bad rebase in a previous patch. The test case was still passing because we expect no route to be reversed..